sormas-app/app/src/main/java/de/symeda/sormas/app/backend/common/DatabaseHelper.java

```diff
@@ -1405,7 +1405,9 @@ public void onUpgrade(SQLiteDatabase db, ConnectionSource connectionSource, int
 				db.execSQL("DROP TABLE tmp_events;");
 
 				getDao(Event.class).executeRaw("UPDATE events set srcType='HOTLINE_PERSON' where length(ifnull(srcFirstName,'')||ifnull(srcLastName,'')||ifnull(srcTelNo,'')||ifnull(srcEmail,'')) > 0;");
-
+				case 210:
+					currentVersion = 210;
+					getDao(Sample.class).executeRaw("ALTER TABLE contacts ADD COLUMN edpidata_id bigint REFERENCES contacts (id);");
```

Comment:
Typo; edpidata_id should be epiData_id and probably reference epidata instead of contacts